internal/security/secretstore/tls/cert.go

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package tls
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func createCA(commonName, certPath, keyPath string) error {
```

Comment:
Allow users to pass all the information, and use default velues if they are empty